custom_components/weenect/sensor.py

```diff
@@ -120,11 +194,33 @@ def async_add_sensors(
 class WeenectSensor(WeenectEntity, SensorEntity):
     """weenect sensor for general information."""
 
+    def _get_call_available(self) -> int:
```

Comment:
```suggestion
    def _get_call_available(self) -> int | None:
```

forgot about the `return None` at the end. My bad